.buildkite/ci.mjs

```diff
@@ -508,14 +484,7 @@ function getBuildCppStep(platform, options) {
  */
 function getBuildToolchain(target) {
   const { os, arch, abi, baseline } = target;
-  let key = `${os}-${arch}`;
-  if (abi) {
-    key += `-${abi}`;
-  }
-  if (baseline) {
-    key += "-baseline";
-  }
-  return key;
+  return [`${os}-${arch}`, abi, baseline].filter(Boolean).join("-");
```

Comment:
can this be reverted?

Reply:
Alright, if that's what it takes to merge, I don't really care.

I find my solution significantly cleaner, but sure

Reply:
I would appreciate some reasoning why, but I've reverted the change

Reply:
What do you mean by _less clever_?

Should we also do `%s/ /\r/g` across all our C/C++/Zig code so that the diff is even cleaner?

The latter is rhetorical, the former is genuinely true. If you really want an even cleaner diff how this

```typescript
return [
  `${os}-${arch}`,
  abi,
  baseline,
].filter(Boolean).join("-");
```

It sounds to me like _less clever_ means _non-functional_. https://en.wikipedia.org/wiki/Structure_and_Interpretation_of_Computer_Programs This is an excellent book I would recommend reading for an introduction to FP (and computing in general) and how FP can improve the structure of code.

One could also argue that _less clever_ means _I didn't like it_, but I'll try my best to avoid taking that interpretation